primitives/runtime/src/calls.rs

```diff
@@ -28,3 +30,21 @@ pub enum SystemCall {
 	#[codec(index = 1)]
 	remark(Vec<u8>),
 }
+
+/// A minimized version of `pallet-sudo::Call` that can be used without a runtime.
+#[derive(Encode, Decode, Debug, PartialEq, Eq, Clone, TypeInfo)]
+#[allow(non_camel_case_types)]
+pub enum SudoCall<Call> {
+	/// `pallet-sudo::Call::sudo`
+	#[codec(index = 0)]
+	sudo(Box<Call>),
+}
+
+/// A minimized version of `pallet-xcm::Call`, that can be used without a runtime.
+#[derive(Encode, Decode, Debug, PartialEq, Eq, Clone, TypeInfo)]
+#[allow(non_camel_case_types)]
+pub enum XcmCall {
```

Comment:
I think `primites/runtime` is more like support for building runtimes,
and this XcmCall/SudoCall is used by relay level stuff,

what about moving it from here under some `relay` directory/package?
probably, this is related to `relay/client-*` layer

Reply:
Yes, I wasn't sure what would be the best place where to define these calls. Moved them, to `client-substrate`.

Reply:
very nice now :)

and also that issue with that "version guard", which means you will need to add `RuntimeVersion` cfg for BridgeHubs, those also should go the the `relay/client-*` level as we discussed "somewhere" with Slava